### PR TITLE
Add a changelog entry on `Protobuf` removal.

### PR DESCRIPTION
**Pull request by var-const**
Removed in #4189.

### CODE REVIEW

**Comment by var-const** [comment]
@wilhuff Seems like something worth calling out. What do you think?

**Comment by var-const** [comment]
@wilhuff The [Travis failure](https://travis-ci.org/firebase/firebase-ios-sdk/jobs/609032369?utm_medium=notification&utm_source=github_status) doesn't make sense to me. Also, when I run the linter locally, it has no complaints. Any ideas?

**Comment by paulb777** [comment]
Trailing space ...

Screen Shot 2019-11-07 at 7 07 15 PM

File: Firestore/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Unreleased
+- [changed] If you're manually tracking dependencies, Firestore SDK no longer
```
Comment (wilhuff) [inline Firestore/CHANGELOG.md]: This phrasing hides what actually changed. What changed should be first and how to potentially handle it should come after.

How about:

- [changed] Removed the Firestore's dependency on `Protobuf`. If you're manually tracking dependencies you may be able to remove it from your build (note, however, that other Firebase components may still require it).

Comment (var-const) [reply Firestore/CHANGELOG.md]: Done.

**Comment by var-const** [comment]
Paul, thanks for spotting this! Looks like my Vim isn't properly configured for Markdown files.

**Pull request merged**